app/models/katello/glue/pulp/consumer.rb

```diff
@@ -188,7 +188,7 @@ def install_consumer_errata(errata_ids)
       end
 
       def katello_agent_installed?
-        return false if self.is_a? Hypervisor
+        return false if (self.is_a? Hypervisor) || @new_record_before_save
```

Comment:
Calling simple_packages causes runcible to throw an error, which is what I'm trying to bypass.